public/index.html

```diff
@@ -9,7 +9,6 @@
       name="description"
       content="Web site created using create-react-app"
     />
-    <link rel="apple-touch-icon" href="%PUBLIC_URL%/logo192.png" />
     <!--
       manifest.json provides metadata used when your web app is installed on a
       user's mobile device or desktop. See https://developers.google.com/web/fundamentals/web-app-manifest/
```

Comment:
Could we please remove the comments ?

Reply:
These are react-generated comments.

Reply:
If these are no longer needed, we can remove those.

Reply:
Please resolve if the comments are needed in future.

Reply:
Please resolve the conversation if these comments are needed in future.